src/main/java/io/github/nucleuspowered/nucleus/modules/ban/commands/TempBanCommand.java

```diff
@@ -39,6 +42,13 @@
     private final String duration = "duration";
 
     @Override
+    public Map<String, PermissionInformation> permissionSuffixesToRegister() {
+        Map<String, PermissionInformation> m = new HashMap<>();
+        m.put("offline", new PermissionInformation(Util.getMessageWithFormat("permission.tempban.offline"), SuggestedLevel.ADMIN));
```

Comment:
Suggested level of `MOD` please - the suggested default should be to allow mods to ban/mute/jail those who are online or offline, and let those who decide otherwise to remove this permission.